3-extra/1-radio-stations.js

```diff
@@ -25,7 +32,17 @@
  * - Return only the frequencies that are radio stations.
  */
 // `getStations` goes here
-
+function getStations(){
+    const allFrequencies = getAllFrequencies();
+    const radioStations = [];
+    for (let i = 0; i < allFrequencies.length; i++) {
+      const frequency = allFrequencies[i];
+      if (isRadioStation(frequency)) {
+        radioStations.push(frequency);
+      }
+    }
```

Comment:
This works fine, but worth looking into `.filter` method in javascript. 
(This will come later in the course, so only if you want a little bit of a challenge)